package.json

```diff
@@ -23,7 +23,8 @@
     "test": "node test/node/*.js && zuul test/browser/*.js"
   },
   "dependencies": {
-    "inherits": "2.0.1"
+    "inherits": "2.0.1",
+    "pify": "^3.0.0"
```

Comment:
This is not the correct way to specify dependencies. All dependencies should be specified using their exact version. Please follow the existing pattern in a file when making changes.